samples/system-test/test.samples.js

```diff
@@ -37,6 +38,13 @@ describe('Cloud Task Sample Tests', () => {
     assert.ok(stdout.includes('Created task'));
   });
 
+  it('should create an HTTP task', async () => {
+    const {stdout} = await exec(
+      `node createHttpTask --project=${PROJECT_ID} --location=us-central1 --queue=${queueName} --url=${url}`
+    );
+    assert.ok(stdout.includes('Created task'));
```

Comment:
It's better to use `match`:

```js
const {assert} = require('chai');
...
assert.match(stdout, /Created task/);
```

That way, if the test fails it will show you what text it tried to match instead of `TRUE !== FALSE`.  You *may* need to `npm install --save-dev chai` for this one to work.